SlicerCART/documentation/installation.md

```diff
@@ -0,0 +1,71 @@
+# SlicerCART Installation
+
+## Requirements
+
+* MacOS Sonoma or Sequoia (15.0.1) is recommended 
+* A working version of [3D Slicer](https://download.slicer.org) (version 5.6.2).
+* Qt: might need to be installed.
```

Comment:
@sandrinebedard from your comments "2- Can you open the python console and screenshot the line where there is the error? (there should have some errors when a slicer module cannot be instantiated).

same thing again, here is the error in the python console:"